### DISCUSSION TITLE
Code for `di`(rectory) Not Working

### DISCUSSION THREAD

**Discussion by snuggs** [eza-community/eza#966] [Q&A, Howto, Help]
Would anyone happen to know the code to use for changing the color of a wildcard?
```
$ exa foo/hel*.wav
```
Need to know the code for the `foo/` part. 
Have tried using the `di`_(rectory)_ code but to no avail.

/cc @ogham
![Screenshot_20240307-084022_Termux](https://github.com/eza-community/eza/assets/38223/1af1597f-ce83-4283-af5a-855c3542cea7)

Please see below:

THANK YOU!

**Comment by daviessm** [comment]
What is the exact command you're running and what environment variables are you setting? In your description you suggest you're running `exa` but this project is `eza`. Are you using an alias?

**Comment by snuggs** [comment]
> What is the exact command you're running and what environment variables are you setting? In your description you suggest you're running `exa` but this project is `eza`. Are you using an alias?

Yes I realized there was a fork and started transitioning the day I made this issue. Converted to `eza` install and changed nothing _(Down to the `EXA_` env variable)_

Same color on both distros. Can't seem to find where to control the subsection.
